go.mod

```diff
@@ -2,8 +2,8 @@ module github.com/codeready-toolchain/host-operator
 
 require (
 	cloud.google.com/go/recaptchaenterprise/v2 v2.13.0
-	github.com/codeready-toolchain/api v0.0.0-20240618040621-92c60519f393
-	github.com/codeready-toolchain/toolchain-common v0.0.0-20240613121043-7e6ef858cdff
+	github.com/codeready-toolchain/api v0.0.0-20240620205422-a29b6c658d03
+	github.com/codeready-toolchain/toolchain-common v0.0.0-20240621204413-d3f78a78634d
```

Comment:
btw, do you know that it's not using the latest version of toolchain-common to promote the change you just merged?